src/ui/components/NetworkMonitor/NetworkMonitorListRow.module.css

```
@@ -37,10 +37,10 @@
 .Row[data-selected]:focus,
 .Row[data-selected]:hover {
   background: var(--primary-accent-hover);
-  color: var(--color-default);
+  color: #fff;
 
-  --status-background-color: var(--color-default);
-  --status-color: var(--color-default);
+  --status-background-color: #fff;
+  --status-color: #fff;
```

This change looks like it would make both background and foreground colors white? That doesn't seem right.

Probably `--theme-selection-background` and `--theme-selection-color` are the variables we should be using here.

```suggestion
  --status-background-color: var(--theme-selection-background);
  --status-color: var(--theme-selection-color);
```